docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -92,43 +94,52 @@ spec:
     ports:
       - containerPort: 443
 ```
-When a pod is attached to a certain network, it needs to set its "network" field:
+When a pod is attached to a certain network, it needs to set its "network" in labels:
 
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
   name: nginx
+  labels:
+    arktos.futurewei.com/network: vpc-1
 spec:
   containers:
   - name: nginx
     image: nginx
     ports:
       - containerPort: 443
-  network: vpc-1
 ```
 
 
-When a pod is attached to a certain network and it wants to specify subnet or IP, it needs to be specified in pod fields:
+When a pod is attached to a certain network and it wants to specify subnet or IP, it can be expressed in annotations:
 
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
   name: nginx
+  labels:
+    arktos.futurewei.com/network: vpc-1
+  annotations:
+    arktos.futurewei.com/nic: {"name": "eth0", "ip": "192.168.0.12"}
 spec:
   containers:
   - name: nginx
     image: nginx
     ports:
       - containerPort: 443
-  network: vpc-1
-  nics:
-    - subnet: subnet-1
-      ip: 192.168.0.12
 ```
 
-If these settings are set on a pod attached to a flat network, the settings will be ignored by the flat network controller and also the corresponding CNI plugins. 
+Annotation arktos.futurewei.com/nic is for user to provide optional information about pod nic. The recognized keys of element include:
+
+|key|exemplary value|
+|---:|---:|
+|name|eth0|
+|subnet|subnet-1, or 192.168.100.0/26|
+|ip|192.168.100.5|
```

Comment:
Do we need to support portID?

Reply:
portID seems neutron specific cni-args. It could be annotated by neutron operator as ani-args, and neutron cni plugin will take podID to wire up nic. 
in previous design, portID and others are all in pod.spec, which serve for two purposes: user hint, extra cni-arg. In this design, it uses two annotations in attempt to separate them.